forester/tests/nullifier_test.rs

```diff
@@ -87,15 +91,15 @@ async fn tree_info_test() {
 #[tokio::test(flavor = "multi_thread", worker_threads = 4)]
 #[ignore]
 async fn test_nullify_leaves() {
-    let indexer = PhotonIndexer::new(INDEXER_URL.to_string());
+    let mut indexer = PhotonIndexer::new(INDEXER_URL.to_string());
     let config = test_config();
-    let client = get_rpc_client_confirmed();
-    client
-        .request_airdrop(&config.payer_keypair.pubkey(), LAMPORTS_PER_SOL * 1000)
-        .unwrap();
+    let mut rpc = SolanaRpcConnection::new(None);
+    let _ = rpc
+        .airdrop_lamports(&config.payer_keypair.pubkey(), LAMPORTS_PER_SOL * 1000)
+        .await;
```

Comment:
same here why not throw an error if the airdrop fails?

Reply:
Fixed.